Doc/library/functools.rst

```diff
@@ -358,6 +378,20 @@ The :mod:`functools` module defines the following functions:
       >>> basetwo('10010')
       18
 
+   If ``Placeholder`` sentinels are present in *args*, they will be filled first
+   when :func:`partial` is called. This allows custom selection of positional arguments
+   to be pre-filled when constructing :ref:`partial object<partial-objects>`.
+   If ``Placeholder`` sentinels are used, all of them must be filled at call time.:
+
+      >>> from functools import partial, Placeholder
```

Comment:
I personally prefer having `PLACEHOLDER` since it's more a module constant rather than a global constant like `None`. In addition, I feel like it would stress the fact that it is a constant and not a class.

Reply:
It kinda follows the pattern of `None`, even if it is not in global namespace. However, I do uppercase my sentinels personally. If there are no strong opinions, I would leave it as it is. After all, none of python's constants of this type are uppercased.

But I am open here if there is something that indicates uppercased would be better.

Reply:
In light of your comment (https://github.com/python/cpython/pull/119827#issuecomment-2155989107), let's keep the PascalCase form.